crates/sui-pg-db/src/lib.rs

```diff
@@ -44,6 +44,14 @@ impl DbArgs {
     pub fn connection_timeout(&self) -> Duration {
         Duration::from_millis(self.connection_timeout_ms)
     }
+
+    pub fn new(database_url: Url) -> Self {
+        Self {
+            database_url,
+            connection_pool_size: 10,
+            connection_timeout_ms: 1000,
+        }
+    }
```

Comment:
Please just use `DbArgs { database_url, ..Default::default() }` i.e. don't add constructors to config types that partially fill them out -- it is exactly the slippery slope that causes us to have multiple notions of "default value".